src/pages/spawn/spawnHost/SpawnHostCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { Link } from "react-router-dom";
+import { SiderCard } from "components/styles";
+import { routes } from "constants/routes";
+import { Host } from "gql/generated/types";
+import { getDateCopy } from "utils/string";
+
+interface SpawnHostCardProps {
+  host: Host;
+}
+
+export const SpawnHostCard: React.FC<SpawnHostCardProps> = ({ host }) => (
+  <StyledSiderCard data-cy="spawn-host-card">
+    {Object.keys(spawnHostCardFieldMaps).map((key) => (
+      <SpawnHostEntry
+        key={`${key}_${spawnHostCardFieldMaps[key](host)}`}
+        field={key}
+        value={spawnHostCardFieldMaps[key](host)}
+      />
+    ))}
+  </StyledSiderCard>
+);
+
+interface SpawnHostEntryProps {
+  field: string;
+  value: string | { key: string; value: string }[];
+  type?: string;
```

Comment:
the type field isn't utilized

Reply:
Thanks i was gonna use it for something but went with a different approach